go-controller/pkg/ovn/loadbalancer.go

```diff
@@ -82,45 +84,173 @@ func (ovn *Controller) getLoadBalancerVIPS(
 	return raw, nil
 }
 
+// deleteLoadBalancerVIP removes the VIP as well as any reject ACLs associated to the LB
 func (ovn *Controller) deleteLoadBalancerVIP(loadBalancer, vip string) {
 	vipQuotes := fmt.Sprintf("\"%s\"", vip)
-	stdout, stderr, err := util.RunOVNNbctl("--if-exists", "remove",
-		"load_balancer", loadBalancer, "vips", vipQuotes)
+	stdout, stderr, err := util.RunOVNNbctl("--if-exists", "remove", "load_balancer", loadBalancer, "vips", vipQuotes)
 	if err != nil {
 		klog.Errorf("Error in deleting load balancer vip %s for %s"+
 			"stdout: %q, stderr: %q, error: %v",
 			vip, loadBalancer, stdout, stderr, err)
+		// if we hit an error and fail to remove load balancer, we skip removing the rejectACL
+		return
+	}
+	ovn.removeServiceEndpoints(loadBalancer, vip)
+	ovn.deleteLoadBalancerRejectACL(loadBalancer, vip)
+	ovn.removeServiceLB(loadBalancer, vip)
+}
+
+func (ovn *Controller) configureLoadBalancer(lb, serviceIP string, port int32, endpoints []string) error {
+	ovn.serviceLBLock.Lock()
+	defer ovn.serviceLBLock.Unlock()
+	commaSeparatedEps := strings.Join(endpoints, ",")
+	target := fmt.Sprintf(`vips:"%s"="%s"`, util.JoinHostPortInt32(serviceIP, port), commaSeparatedEps)
+
+	out, stderr, err := util.RunOVNNbctl("set", "load_balancer", lb,
```

Comment:
@trozet may as well move target to the end of the previous line since its no longer anywhere near the longest line in the function.

Reply:
ok